app/src/main/res/values-tr/strings.xml

```diff
@@ -0,0 +1,24 @@
+<resources>
+    <string name="app_name">SauceNAO</string>
```

Comment:
I think it's better to mark this as translatable="false" in values/strings.xml.

Reply:
Did that here: 1ab1f1af8ea489eafc9c96a5d7a584aac4ec5f0d

Reply:
OK, i have pushed a new commit which removes that line and also updates it in the values/strings.xml file.